lib/cocoapods/bazel/target.rb

```diff
@@ -317,6 +332,40 @@ def defaults
         }
       end
 
+      def deps_by_config
+        debug_targets = dependent_targets_by_config[:debug]
+        release_targets = dependent_targets_by_config[:release]
+
+        debug_labels = debug_targets.map { |dt| dt.bazel_label(relative_to: package) }
+        release_labels = release_targets.map { |dt| dt.bazel_label(relative_to: package) }
+        shared_labels = (debug_labels & release_labels).uniq
+
+        debug_only_labels = debug_labels.reject { |x| shared_labels.include? x }
+        release_only_labels = release_labels.reject { |x| shared_labels.include? x }
+
+        sorted_debug_labels = Pod::Bazel::Util.sort_labels(debug_only_labels)
+        sorted_release_labels = Pod::Bazel::Util.sort_labels(release_only_labels)
+        sorted_shared_labels = Pod::Bazel::Util.sort_labels(shared_labels)
+
+        labels_by_config = {}
+
+        if !sorted_debug_labels.empty? || !sorted_release_labels.empty?
+          labels_by_config['//Pods/cocoapods-bazel:debug'] = sorted_debug_labels
```

Comment:
I dont think we should assume the `Pods` directory is always named `Pods` , see how we write to the `sandbox.root` ?